app/frontend/src/features/communities/events/EventsSection.tsx

```diff
@@ -0,0 +1,74 @@
+import { CircularProgress, Link as MuiLink } from "@material-ui/core";
+import Alert from "components/Alert";
+import HorizontalScroller from "components/HorizontalScroller";
+import { CalendarIcon } from "components/Icons";
+import TextBody from "components/TextBody";
+import {
+  EVENTS_EMPTY_STATE,
+  EVENTS_TITLE,
+  SHOW_ALL_EVENTS,
+} from "features/communities/constants";
+import { Community } from "proto/communities_pb";
+import { Link } from "react-router-dom";
+import { routeToCommunity } from "routes";
+import hasAtLeastOnePage from "utils/hasAtLeastOnePage";
+
+import { SectionTitle, useCommunityPageStyles } from "../CommunityPage";
+import { useListCommunityEvents } from "../hooks";
+import EventCard from "./EventCard";
+
+export default function EventsSection({
+  community,
+}: {
+  community: Community.AsObject;
+}) {
+  const classes = useCommunityPageStyles();
+
+  const { data, error, hasNextPage, isLoading } = useListCommunityEvents({
+    communityId: community.communityId,
+    pageSize: 3,
```

Comment:
Do 6 - divisible by 3 and 2 (for tablets)

Reply:
From the design, don't we only want to show one row of events?

Reply:
Yeah probably, it looks weird on tablets though.